README.md

```diff
@@ -120,6 +120,7 @@ UI tests require credentials that are only available to Twilio employees.
 
 ## Related
 
+- [Change Log](CHANGELOG.md)
```

Comment:
I could see adding a `## Change Log` section above related section. I think the intent of related is to link related repos together.

But I'm sure after the meeting Friday we'll get things how the team wants them to be in any case.

Reply:
We can cut this part for now if we aren't sure its appropriate.